### ISSUE TITLE
Autoprefixer processing comment to locally turn off grid polyfill continues to report IE related warnings

### ISSUE THREAD

**Issue by rjgotten** [postcss/autoprefixer#1333]
Given the following snippit of CSS

```css
.usps > ul {
  display : flex; 
  // ...
}

@supports(grid-template-columns: repeat(auto-fit, minmax(200px, max-content))) {
  .usps > ul {
    /* autoprefixer grid: off */
    display               : grid;
    grid-gap              : 20px 10px;
    grid-template-columns : repeat(auto-fit, minmax(200px, max-content));
    justify-content       : center;
  }
}
```

Even though it is using the `/* autoprefixer grid: off */` processing comment, this particular block continues to raise a warning that " `IE does not support justify-content on grid containers` ." when conditionally applying grid styling.

Elevating the processing comment to the scope of the `@supports` block itself yields the same result. It continues to report warnings related to IE's grid implementation. Only using `/* autoprefixer: off */` in these places works as expected.

While that suffices in the current situation for my problem, it is highly non-desired that the entirety of Autoprefixer has to be turned off. This would start causing problems once these rules end up containing properties which _do_ need prefixing, requiring that e.g. an artificially split CSS rule be maintained where autoprefixer can still be turned off.

**Autoprefixer version** : 9.8.4

**Comment by ai** [comment]
Thanks for the issue.

Do you want to send a pull request to save your name in the project history? You will need to add `this.gridStatus(decl, result)` check before [the warning](https://github.com/postcss/autoprefixer/blob/33ee9897b1eaa591ed3bb80bb098b29fbed5eab7/lib/processor.js#L150).

**Comment by rjgotten** [comment]
@ai 
Nah; I'm good as is. I just filed the issue; _you_ actually found the fix. :+1:

**Comment by ai** [comment]
I will be able to fix it only after a few weeks. Too much work too.

**Comment by rjgotten** [comment]
When it rains it pours, right? ;)

Anyway; the current work-arounds I have will hold for the short term, so a few weeks is fine.
A quick guessstimate is that a few weeks would be the time window before I could start on a PR as well.

If for some reason I find the time to work on it sooner, can I notify you through this issue?

**Comment by ai** [comment]
I am preparing PostCSS 8 right now. This release could take 2-4 weeks.

**Comment by ai** [comment]
The fix was released in 9.8.5.
